.github/workflows/publish-docker.yml

```diff
@@ -9,6 +9,9 @@ on:
       gcrJsonKey:
         description: "JSON key for GCR authentication"
         required: true
+      dockerSecrets:
+        description: "Docker build push action"
+        required: false
```

Comment:
not sure what happens if a non-required field doesn't have a default set, so please confirm that it is still working without a dockerSecrets param set

Reply:
Secrets doesn't support defaults, only inputs. Since they do support require: false I am fairly confident that this should work - but will verify after merging.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callsecrets